sentences/fr/homeassistant_HassNevermind.yaml

```diff
@@ -7,6 +7,6 @@ intents:
           - "annule[r]"
           - "[non] rien [du tout]"
           - "chut"
-          - "tais toi"
-          - "tais-toi"
+          - "tais(-| )toi"
```

Comment:
Please keep it a 2 separate sentences as it can be hard for beginner to understand this syntax.